src/arbitraries/Arbitrary.ts

```diff
@@ -47,7 +45,7 @@ export abstract class Arbitrary<A> {
    * Returns a sample of picks of a given size withour replacement. Sample will
    * not contain repeated values. Corner cases are not taken into account.
    */
-  sampleUnique(sampleSize = 10, cornerCases: FluentPick<A>[] = []): FluentPick<A>[] {
+  sampleUnique(sampleSize = 10, generator: () => number, cornerCases: FluentPick<A>[] = []): FluentPick<A>[] {
```

Comment:
Please provide a default generator _and_ probably make it the last argument... arguments _with defaults_ should be ordered by likelihood of having an argument passed, and in case of a tie, in the end so as to avoid changing unnecessary code.